### DISCUSSION TITLE
Authenticate Anonymously

### DISCUSSION THREAD

**Discussion by fabius** [zitadel/zitadel#5574] [Ideas]
As a user, I would like to be able to first authenticate anonymously / with a temporary account to first give an application a try. This temporary account should have all credentials created and stored in my cookies so that there's zero manual input from my side. If I decide to create an "actual" account later on, the temporary account should be converted. Anonymous accounts could optionally, for example, be auto deleted after {x timespan}. Some setting for this could be provided in the instance/organization settings.

[This one of Firebase's features](https://firebase.google.com/docs/auth/web/anonymous-auth) that I dearly miss in Zitadel

**Comment by fforootd** [comment]
Our @peintnermax brought this up as well. While we are creating our "new" login API we will work on this feature. Can't yet promise a ETA yet though.

#4193

**Comment by cortopy** [comment]
@fforootd I checked the issue you mention and can’t find any information there about anonymous sessions. Is this something zitadel is still looking at?
